airbyte-cdk/python/unit_tests/connector_builder/test_connector_builder_handler.py

```diff
@@ -704,10 +706,10 @@ def _create_response(body, request):
 
 def _create_page(response_body):
```

Comment:
nit: I think this method should be renamed since it returns a Response, not a pair of request,response